frank/default_parameters.json

```diff
@@ -11,7 +12,10 @@
 
   "modify_data" : {
     "cut_data"         : false,
-    "cut_baseline"     : null
+    "cut_range"        : null,
```

Comment:
There are two parameters being used where a single one if sufficient. Also, be more explicit in the name, something like "baseline_range" would suffice